mlir/utils/performance/perfRunner.py

```diff
@@ -237,7 +237,8 @@ def getBankConflict(fileName):
 
 # Tuning databases
 MaybeTuningDb = Optional[Dict[Tuple[str, str], str]]
-def read_tuning_db(path: Optional[str]) -> MaybeTuningDb:
+MaybeTuningDbWithCU = Optional[Dict[Tuple[str, str, str], str]]
```

Comment:
nit:
I don't particularly like keeping unamed tuple/lists. it is error prone and not not very readable. It is better to use dataclass/struct/classes etc. 
Something to be improved.